### ISSUE TITLE
Scrolling lags on Firefox OS 2.5

### ISSUE THREAD

**Issue by floppyrom** [minego/macaw-enyo#79]
Scrolling in Macaw on my Flame device running Firefox Os 2.5 is not smooth at all.

**Comment by soapdog** [comment]
I have it here on the Sony Z3C with Firefox OS 2.5, it scrolls well. I believe it may be related to the flame port. Have you tried it with 1.3 on flame?

**Comment by floppyrom** [comment]
No. I can't downgrade now. It just feels that it's running at 15fps when compared with the official Twitter app that runs at 30 fps :)

**Comment by soapdog** [comment]
Can you double check your developer settings? there are some settings regarding hardware composer, tiling and scrolling there. I will try on a flame here next week and get back. Maybe I can reproduce and we can get a recipe...

**Comment by floppyrom** [comment]
Awesome! 

Meanwhile I checked and I have by default: Overscolling, Tiling, low precision painting, low precision transparency, hardware composer all checked
